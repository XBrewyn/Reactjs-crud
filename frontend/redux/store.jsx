import { createStore, compose, applyMiddleware } from 'redux';
import { devToolsEnhancer } from 'redux-devtools-extension/developmentOnly';
import thunk from 'redux-thunk';
import reducer from './reducers';

const store = createStore(
  reducer,
  compose(applyMiddleware(thunk), devToolsEnhancer())
);

export default store;
