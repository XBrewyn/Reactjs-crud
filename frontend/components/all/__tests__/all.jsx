import React from 'react';
import { create } from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../../redux/store';
import All from '../all';

describe('<All />', () => {
  it('renders correctly', () => {
    const tree = create(
      <Provider store={store}>
        <All />
      </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
