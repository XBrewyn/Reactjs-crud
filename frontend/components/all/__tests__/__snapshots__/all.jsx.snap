// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<All /> renders correctly 1`] = `
<div
  className="css-1b9l8d"
>
  <h1>
    All
  </h1>
  <table>
    <thead>
      <tr>
        <th>
          id
        </th>
        <th>
          name
        </th>
        <th>
          password
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
